python/src/opendp/measurements.py

```diff
@@ -123,10 +123,19 @@ def then_alp_queryable(
     size_factor = 50,
     alpha = 4,
     CO: Optional[RuntimeTypeDescriptor] = None
-):
-    r"""Partial constructructor which uses the preceding output domain and metric as its input domain and metric. See also:
+):  
+    r"""partial constructor of make_alp_queryable
 
-    :py:func:`opendp.measurements.make_alp_queryable`
+    .. seealso:: 
+      Delays application of `input_domain` and `input_metric` in :py:func:`opendp.measurements.make_alp_queryable`
+
+    :param scale: Privacy loss parameter. This is equal to epsilon/sensitivity.
+    :param total_limit: Either the true value or an upper bound estimate of the sum of all values in the input.
+    :param value_limit: Upper bound on individual values (referred to as β). Entries above β are clamped.
+    :param size_factor: Optional multiplier (default of 50) for setting the size of the projection.
+    :param alpha: Optional parameter (default of 4) for scaling and determining p in randomized response step.
+    :param CO: 
```

Comment:
add description, particularly since the name may not be meaningful to someone who starts here.